domains/emissions/Emissions.API/API/Models/EmissionsResponse.cs

```diff
@@ -22,12 +22,12 @@ public class Emissions
     public long DateTo { get; }
 
     [JsonPropertyName("total")]

```

Comment:
Are these attributes necessary for serialization? Can't casing be configured?

Reply:
Attributes deleted